packages/core/src/components/StepList/StepLink.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class StepLink extends React.PureComponent {
+  handleClick(event) {
+    if (this.props.onEnterStep) {
+      event.preventDefault();
+      this.props.onEnterStep(this.props.route, this.props.stepId);
+    }
+  }
+
+  render() {
+    const { route, screenReaderText, className, children } = this.props;
+    const onClick = event => this.handleClick(event);
+    return (
+      <a href={route} onClick={onClick} className={className}>
+        {children}
+        <span className="ds-u-visibility--screen-reader">
+          {' '}
+          {screenReaderText}
+        </span>
+      </a>
+    );
+  }
+}
+
+StepLink.propTypes = {
```

Comment:
I'd love to see these documented on the docs page – I think some people (including myself) would find it valuable.